contactsummary/js/src/contact-summary/services/contract.service.js

```diff
@@ -174,6 +175,22 @@ define([
     }
 
     /**
+     * Returns the contract field options
+     *
+     * @param  {string} fieldName
+     * @return {Promise}
+     */
+    function getOptions (fieldName) {
+      var options = settings.CRM.options.HRJobDetails || {};
+
+      if (fieldName && typeof fieldName === 'string') {
```

Comment:
Do we really need `fieldName && `?  I think `if (typeof fieldName === 'string')` should be enough.